web/email.ex

```diff
@@ -5,46 +5,50 @@ defmodule Community.Email do
   def admin_job_posted(job) do
     new_email()
     |> to(admin_emails())
-    |> from("noreply@raleighdesign.io")
+    |> from(noreply_email())
     |> subject("A new job has been posted")
     |> render("admin_job_posted.text", job: job)
   end
 
   def job_posted(job) do
     new_email()
     |> to(job.contact)
-    |> from("noreply@raleighdesign.io")
+    |> from(noreply_email())
     |> subject("Thank you for posting")
     |> render("job_posted.text", job: job)
   end
 
   def admin_member_added(member) do
     new_email()
     |> to(admin_emails())
-    |> from("noreply@raleighdesign.io")
+    |> from(noreply_email())
     |> subject("A new member has signed up!")
     |> render("admin_member_added.text", member: member)
   end
 
   def member_added(member) do
     new_email()
     |> to(member.email)
-    |> from("noreply@raleighdesign.io")
+    |> from(noreply_email())
     |> subject("Thank you for signing up")
     |> render("member_added.text", member: member)
   end
 
   def contact_form(data) do
     new_email()
     |> to(data[:to])
-    |> from("noreply@raleighdesign.io")
+    |> from(noreply_email())
     |> put_header("Reply-To", data.from)
-    |> subject("Raleigh Design Contact Form - #{data.subject}")
+    |> subject("Triangle Designers Contact Form - #{data.subject}")
```

Comment:
I see "Triangle Designers" repeated. What do you think of extracting this to a constant somewhere?